src/main/resources/translations/program_strings_en.properties

```
@@ -461,128 +461,127 @@ rph.donotuse = Do not use
 rph.location = location
 rph.cityname = City name
 rph.iso = ISO
-rph.foclength35mm = Focal length in 35mm format
+rph.foclength35mm = Equivalent 35 mm focal length
 rph.orgfilename = Original filename
-rph.duplicatestext = In case your chosen options lead to duplicate names, the tool will automatically autonumber the images.<br>Please specify how the tool should do this.
+rph.duplicatestext = In case your chosen options leads to duplicate names, the tool auto-numbers the images.<br>Please specify how the tool should do this.
 rph.noofdigits = No. of digits
-rph.nooffidgitsone = Start counting on 1st image
-rph.nooffidgitstwo = Start counting on 2nd image
+rph.nooffidgitsone = Start counting from 1st image
+rph.nooffidgitstwo = Start counting from 2nd image
 rph.filextension = File extension:
 rph.fileextleave = Leave as is
-rph.fileextlower = Make lower case
-rph.fileextupper = Make upper case
+rph.fileextlower = Make lowercase
+rph.fileextupper = Make uppercase
 rph.nameexample = Name: Prefix_suffix.extension
-rph.locateimgfolder = Locate the image folder ...
+rph.locateimgfolder = Locate the image folder…
 rph.noimgsnopathtitle = No path or selected files
 rph.noimgsnopathtext = No image folder path selected<br>and/or<br>No images selected in the main screen
 rph.selectedoptionstitle = Selected options
 rph.youselected = You selected:
 rph.asprefix = As prefix:
 rph.assuffix = And as suffix:
-rph.forcounting = For counting:
+rph.forcounting = For enumerating:
```

Tallying?
```suggestion
rph.forcounting = For tallying:
```
(To arrive at one sum, as opposed to assigning numbers.

---

Thanks for making me aware of this. It should be numbering.
When images might get the same name, they are auto numbered as well, based on first to last DateTimeOriginal.